### ISSUE TITLE
[Bug]: Faulty scam warning message

### ISSUE THREAD

**Issue by KenshiHashiguchin** [MetaMask/metamask-extension#24146]
### Describe the bug

One of our customers receives the following advert:

This is a fraudulent request.
If you authorise this request, a third party known for fraud will seize all your assets.

Well, that's very unpleasant for us. 
Our service: https://app.akindo.io

Our service, WaveHack, is an ongoing hackathon program that aims to distribute a total of $500,000 (75 million yen!!) in grants to over 5,000 engineers from April 13th (Saturday) to the end of August. In conclusion, it is a program for developers who have the opportunity to win up to $50,000 every two weeks, and anyone can participate for free.

We already have a track record of distributing grants through the WaveHack hosted by Aptos.
Additionally, the following projects are planned to distribute grants in the future.
- Manta Network
- The Graph
- Mina Protocol
- Superfluid

WaveHack: https://app.akindo.io/wave-hacks

Therefore, our service is not a scam.

### Screenshots/Recordings

https://github.com/MetaMask/metamask-extension/assets/46628255/79a6f83e-7ee8-4245-bd88-112b92386bdd

### Steps to reproduce

1. Akindo Signup (https://app.akindo.io/login)
2. Please enter the required information and click "Sign Up".
3. You will receive an email. Please click on the "Confirm my account" link in the email.
4. Sign in from the "Buidl" option in the top right corner of the screen.
5. Once you have signed in, click on the cat icon in the top right corner of the screen. and Click the "Create Project" button.
6. Please fill in the required information to create the project.
7. Once your project is ready, click on "Open Hackathon" in the top right corner of the screen and select "WaveHack".
8. Please fill in the required information and click "Open".
9. click "Wallet Connect" 

![スクリーンショット 2024-04-20 11 37 21](https://github.com/MetaMask/metamask-extension/assets/46628255/72b4883e-535d-462f-b812-b1f6f5be2040)

10. When the wallet selection screen appears, please select MetaMask.
11. Select Ethereum and choose "Deposit." This action will open MetaMask. At this time, a BlockID alert will be displayed.
![スクリーンショット 2024-04-20 11 38 59](https://github.com/MetaMask/metamask-extension/assets/46628255/85767ca2-cf53-4405-bda9-bab1d2f95d04)

### Version

the latest

### Build type

None

### Browser

Chrome

### Operating system

Windows, MacOS

### Additional context

• The contract 0x0167a6954824f8bd420fb460446e880c7bbccafb involved in the transaction is unverified

**Comment by gauthierpetetin** [comment]
Hello, @KenshiHashiguchin. Thank you for your report. Could you click the "Report an issue" within the warning and submit the form? That's the quickest way to have such issues addressed.

**Comment by KenshiHashiguchin** [comment]
@gauthierpetetin 
thank you! I've already done this several times, but since they haven't responded, I'm making a direct request here.You guys are solving a similar problem here, and it would be helpful if you could do the same.
https://github.com/MetaMask/metamask-extension/issues/22062

**Comment by gauthierpetetin** [comment]
I'm sorry to hear the "Report an issue" solution mentioned in my previous message takes time to respond.
Not sure there's an alternative solution, but I'll let @bschorchit share more detail if that's the case.

**Comment by bschorchit** [comment]
Closing as this has been solved. Let us know if you encounter any issues again.

**Comment by christopheSeeka** [comment]
Same issue, here, very harmfull to get this while we are a fully compliant and registered project... We get this error on BNB Chain only, not on Polygon where we run the exact same dapp. I filled out the form but our public sale is in 2 days, any chance to help us in shorter timeframe?
